src/deviceApiCalls/schemas/incontext-signup-settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "IncontextSignupSettings",
+    "type": "object",
+    "additionalProperties": false,
+    "description": "Delivered as part of Runtime Configuration, but needs to live here since Runtime Configuration can contain settings for many features",
```

Comment:
I'm a little unclear as to why this should be separate.

Reply:
It's because this PR introduces a second 'feature' that can be configured from the remote config, it looks correct to me :)